README.md

```diff
@@ -1,91 +1,135 @@
+- [Computing spatial bounding boxes of trajectories](https://docs.rapids.ai/api/cuspatial/stable/user_guide/cuspatial_api_examples.html#cuspatial.trajectory_bounding_boxes)
+
+### What if operations I need aren't supported?
+Thanks to the `from_geopandas` and `to_geopandas` functions you can accelerate what cuSpatial supports, and leave the rest of the workflow in place:
+
+```mermaid
+---
+title: Integrating into Existing Workflows
+---
+%%{init: { 'logLevel': 'debug', 'theme': 'base', 'gitGraph': {'showBranches': false},
+            'themeVariables': {'commitLabelColor': '#000000',
+            'commitLabelBackground': '#ffffff',
+            'commitLabelFontSize': '14px'}} }%%
+gitGraph
+   commit id: "Existing Workflow Start"
+   commit id: "GeoPandas IO"
+   commit id: "Geospatial Analytics"
+   branch a
+   checkout a
+   commit id: "from_geopandas"
+   commit id: "cuSpatial GPU Acceleration"
+   branch b
+   checkout b
+   commit id: "cuDF"
+   commit id: "cuML"
+   commit id: "cuGraph"
+   checkout a
+   merge b
+   commit id: "to_geopandas"
+   checkout main
+   merge a
+   commit id: "Continue Work"
+```
 
-### Fetch cuSpatial repository
 
-```shell
-export `CUSPATIAL_HOME=$(pwd)/cuspatial` && \
-git clone https://github.com/rapidsai/cuspatial.git $CUSPATIAL_HOME
+## Using cuSpatial
+**CUDA/GPU requirements**
+```
+CUDA 11.2+
+NVIDIA driver 450.80.02+
+Pascal architecture or better (Compute Capability >=6.0)
 ```
-### Install dependencies
 
-1. `export CUSPATIAL_HOME=$(pwd)/cuspatial`
-2. clone the cuSpatial repo
+### Quickstart - Docker
+Use the [RAPIDS Release Selector](https://rapids.ai/start.html#get-rapids), selecting `Docker` as the installation method. All RAPIDS Docker images contain cuSpatial.
 
+An example command from the Release Selector:
 ```shell
-conda env update --file conda/environments/all_cuda-115_arch-x86_64.yaml
+docker pull nvcr.io/nvidia/rapidsai/rapidsai-core:23.02-cuda11.8-runtime-ubuntu22.04-py3.10
+docker run --gpus all --rm -it \
+    --shm-size=1g --ulimit memlock=-1 --ulimit stack=67108864 \
+    -p 8888:8888 -p 8787:8787 -p 8786:8786 \
+    nvcr.io/nvidia/rapidsai/rapidsai-core:23.02-cuda11.8-runtime-ubuntu22.04-py3.10
 ```
 
-### Build and install cuSpatial
+### Install from Conda
 
-1. Compile and install
-   ```shell
-   cd $CUSPATIAL_HOME && \
-   chmod +x ./build.sh && \
-   ./build.sh
-   ```
+To install via conda:
+> **Note** cuSpatial is supported only on Linux or [through WSL](https://rapids.ai/wsl2.html), and with Python versions 3.8 and later
 
-2. Run C++/Python test code
+cuSpatial can be installed with conda (miniconda, or the full Anaconda distribution) from the rapidsai channel:
 
-   Some tests using inline data can be run directly, e.g.:
+```shell
+conda install -c rapidsai -c conda-forge -c nvidia \
+    cuspatial=23.04 python=3.10 cudatoolkit=11.8
```

Comment:
Can we somehow auto-update this line in each release branch?

Reply:
Great question! I'll look into it. Also will be relevant when we switch Python or CUDA versions.

Reply:
I think we can use our [update version script](https://github.com/rapidsai/cuspatial/blob/branch-23.04/ci/release/update-version.sh) to do this, will file an issue to get this done in 23.06 (don't want to delay the readme, and it won't be out of date until 23.06.)